File: Assignment_3/Assignment3_3.py
"""-------------------------------------------------------------------------------------------
                          Assignment3_3
                    (Student name - Vaishali Jorwekar)
                     Python by Marvellous Infosystems
---------------------------------------------------------------------------------------------
Problem statement : This program accepts N elements from the user and 
                    store it into a list and returns the minimum number
                    from the list
------------------------------------------------------------------------------------------"""
from MarvellousFun import acceptElements
#-------------------------------------------------------------------------------------
# This Function finds the minimum number from the list
#-------------------------------------------------------------------------------------
def findMinimumElement(numberList):
    min=numberList[0]
    for num in numberList:
         if num < min :
             min = num
    print("-------------------------------------------------------------------")
    print(f"Minimum element from the list {numberList} is : {min}")     
    print("-------------------------------------------------------------------")

#------------------------------------------------------------------------------------------
# This function accepts a number of elements number from user and calls function 
# acceptElements(number), findMinimumElement(inputNumberList)
#------------------------------------------------------------------------------------------
def main():
    noOfElements=int(input("Enter number of elements in the list:"))
    inputNumberList=acceptElements(noOfElements)
    findMinimumElement(inputNumberList)
#-----------------------------------------------------------------------------------
# This is main entry point of the program
#-----------------------------------------------------------------------------------
if __name__=="__main__":
    main()
#-----------------------------------------------------------------------------------

File: JavaPractice/Assignment_3/Assignment3_3.java
/* -------------------------------------------------------------------------------------------
                          Assignment3_2
                    (Student name - Vaishali Jorwekar)
                     Python by Marvellous Infosystems
---------------------------------------------------------------------------------------------
Problem statement : This program accepts N elements from the user and 
                    store it into a list and returns the Minimum number 
                    from the list
-----------------------------------------------------------------------------------------  */


import java.util.ArrayList;


public class Assignment3_3 {
	
	public static void main(String args[]) {
		
		ArrayList<Integer> numberList=Module.acceptNumbers();
		System.out.println("List Elements are:"+numberList);
		findMinimumNumber(numberList);
		

	
}

	private static void findMinimumNumber(ArrayList<Integer> numberList) {
		int minNumber=numberList.get(0);
		for(int cnt=0;cnt<numberList.size();cnt++) {
			int element=numberList.get(cnt);
			if(minNumber>element) {
				
				minNumber=element;
			}
		}	
		System.out.println("Minimum element is :"+minNumber);
	}

}
